### PR TITLE
3-GET-/api/categories implementation TDD and error handling

### PR DESCRIPTION
**Pull request by Oliver1334**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+});
+
+describe('GET: /api/categories errors', () => {
+            test('responds with an error message to a non-existant endpoint', ()=> {
+               return request(app)
+                    .get('/api/notAnEndpoint')
+                    .expect(404)
+                    .then((res)=>{
+                        console.log(res.body)
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:52]: Make sure to remove console.logs and comments before committing :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            },
+            {
+              slug: 'dexterity',
+              description: 'Games involving physical skill'
+            },
+            {
+              slug: "children's games",
+              description: 'Games suitable for children'
+            }
+          ];
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:34]: Make your test more dynamic, dont hard code values. Think of other ways you can test your result.

**Review by Beth-McCormack — CHANGES_REQUESTED** [review]
see comments

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        //             })
+        //     })
+    
+
+
+
+
+            // a reviews array of review objects, each of which should have the following properties:
+
+            // owner
```
Comment (yalicee) [inline __tests__/app.test.js:62]: delete all comented out code

**Review by yalicee — APPROVED** [review]
Looks really good Oliver!

**Pull request merged**